### PR TITLE
Add account-manager env vars to account-api

### PR DESCRIPTION
**Pull request by barrucadu**
[Trello card](https://trello.com/c/ZeAKl2SM/651-set-up-a-new-app)

### CODE REVIEW

File: hieradata_aws/integration.yaml [hunk 1]
```diff
@@ -10,6 +10,7 @@ cron::daily_hour: 6
 
 environment_ip_prefix: '10.1'
 
+govuk::apps::account_api::plek_account_manager_uri: 'https://www.account.staging.publishing.service.gov.uk'
```
Comment (huwd) [inline hieradata_aws/integration.yaml:13]: Should this be staging or integration?

**Review by huwd — CHANGES_REQUESTED** [review]

File: hieradata_aws/integration.yaml (hunk 1 above)
Comment (huwd) [reply hieradata_aws/integration.yaml:13]:
```suggestion
govuk::apps::account_api::plek_account_manager_uri: 'https://www.account.integration.publishing.service.gov.uk'
```

Comment (barrucadu) [reply hieradata_aws/integration.yaml:13]: This should actually be staging, as we're not running a separate integration account manager - see also finder-frontend's hieradata

Comment (huwd) [reply hieradata_aws/integration.yaml:13]: ok ace, will approve

**Review by huwd — APPROVED** [review]

**Pull request merged**